### DISCUSSION TITLE
Heimdall LXC: 500 | Server Error

### DISCUSSION THREAD

**Discussion by circa1741** [tteck/Proxmox#103] [General]
The latest script yesterday worked and I was able to configure the Heimdall Dashboard. However, this morning, the Heimdall Dashboard is returning "500 | Server Error." I rebooted the LXC but no luck. I even reboot the host too but still no luck.

![image](https://user-images.githubusercontent.com/8772586/160286738-ebe71976-399c-45c9-b5b1-d5b426a71bd6.png)

**Comment by tteck** [comment]
https://github.com/linuxserver/Heimdall/releases/tag/v2.4.8
>Check is class exists before checking if it's a search provider... will hopefully cut down on apps generating error 500
